docs/vgclog_examples.md

```diff
@@ -706,3 +707,895 @@ Analyzing the structure and elements of this example log output shows that this
+  <response-info timems="3.082" idlems="1.041" threads="2" lastid="00000000006EDE00" lastname="RunDataWriter.1" />
+</exclusive-start>
+```
+
+The cycle is triggered when the allocation taxation threshold is reached. The <allocation-taxation> element records the memory threshold value that was in use when this example global mark cycle was triggered. The`taxation-threshold` attribute shows the taxation threshold is 1024MB (1,073,741,824), which is half the total memory of the *eden* space (2048 MB), as expected for threshold triggers of global mark cycles and increments. For more information taxation thresholds for the `balanced` policy, see [`balanced` examples](vgclog_examples.md#balanced-examples).
+
+```xml
+<allocation-taxation id="1153" taxation-threshold="1073741824" timestamp="2021-02-26T11:17:25.034" intervalms="1931.251" />
+```
+
+Details about the start of the global mark GC cycle are recorded by the `<cycle-start>` element. The cycle is recorded as type `global mark phase` with an `id=1154`. Any subsequent elements associated with this cycle have a `contextid=1154` to match the global mark GC cycle's `id`. You can use the `contextid` value to distinguish increments and operations of the global mark GC cycle from the partial cycles that interleave with it.
+
+```xml
+<cycle-start id="1154" type="global mark phase" contextid="0" timestamp="2021-02-26T11:17:25.034" intervalms="374365.075" />
+```
+
+The cycle begins with the STW subincrement of a global *mark* phase increment. The STW subincrement is recorded using the `<gc-start>` element of type `global mark phase`.
+
+```xml
+<gc-start id="1155" type="global mark phase" contextid="1154" timestamp="2021-02-26T11:17:25.034">
+  <mem-info id="1156" free="1442840576" total="4294967296" percent="33">
+    <remembered-set count="2197888" freebytes="162912768" totalbytes="171704320" percent="94" regionsoverflowed="3" regionsstable="130" regionsrebuilding="0"/>
+  </mem-info>
+</gc-start>
+
+```
+
+The `<gc-start>` element provides a snapshot of the free memory available in the heap and the status of marked objects. At the start of the increment, the heap is 33% free, with 1376 MB (1442840576 B) of the total 4096 MB (4294967296 B) heap free.
+
+The `<remembered-set>` element records the status of the remembered set metastructure of the JVM, a structure that records object references that cross different regions. During the rebuilding of the *remembered set* metastructure, any regions that cannot be rebuilt into a remembered set due to a lack of memory resource in the metastructure are marked as *overflow* regions. Cycles cannot reclaim memory from overflow regions.
+
+The aim of the global mark cycle to create a new record of object liveness can be seen in the remembered set statistics. The global mark cycle populates the remembered set and also attempts to completely rebuild the set for the overflowed regions. 
```

Comment:
Rebuilding liveness information and rebuilding overflowed RS are mostly indenpendent processes (they are not consequential). I'd rephrase all this paragraph something like this (I'm adding an extra fact/sentence, too):

The aim of the global mark cycle to create a new record of object liveness, but also to attempt to rebuild overflowed region remembered sets. By the end of the GMP cycle some of those remembered sets may again overflow, but some may be successfully rebuilt. After the cycle is finished, based on more precise liveness information, subsequent PGCs will also be able to prune remembered sets (those that were not overflowed) even more (precisely or thoroughly, or just no need for any adverb here).

Reply:
Modified to:

"The aim of the global mark cycle is to create a new record of object liveness by populating the remembered set. The global mark cycle also attempts to rebuild the remembered set information for the overflowed regions, which can be seen in the remembered set statistics. After the global mark cycle completes, the remembered set reflects a closer snapshot of the current liveness of the heap. This closer snapshot of object liveness optimizes the pruning of the set, which is executed by the partial GC cycle when it consumes the object liveness snapshot."